admin/authentication/providers.md

```diff
@@ -0,0 +1,21 @@
+# Manage authentication providers
+
+Stoplight supports the following authentication providers:
+
+- Username and password
+- LDAP
+- <!-- QUERY: which others? -->
+
+## Add a new provider
+
+1. Select **Add Provider**.
+2. Provide the details of the new service, including:
+    1. **Name**: the name for this provider in Stoplight.
+    2. **URL**: <!-- QUERY: more description for each of these? -->
+    3. **Bind DN**:
+    4. **Bind credentials**:
+    5. **Search base**:
+    6. **Search filter**:
+3. Select **Save**.
+
+<!-- TODO: continue once this functionality is further along. At present seems very LDAP-focused? -->
```

Comment:
It is very LDAP focused :) All we support is LDAP SSO and email/password basic auth.